tests/api.c

```diff
@@ -9658,7 +9679,7 @@ static void test_wolfSSL_ERR_peek_last_error_line(void)
     AssertNotNull(file);
 
 #ifdef WOLFSSL_TIRTOS
-    fdOpenSession(Task_self());
+   fdOpenSession(Task_self());
```

Comment:
Please revert this indent change. Should be 4 spaces, per our coding standard.

Reply:
I fixed.